fabric-configurator/src/main/java/com/luxoft/fabric/config/NetworkManager.java

```diff
@@ -424,4 +427,33 @@ private static void installChaincodes(HFClient hfc, FabricConfig fabricConfig, L
             }
         }
     }
+
+
+    @SuppressWarnings("unused")
+    public static void deployChaincode(FabricConnectorImplBasedOnFabricConfig fabricConnector, String chaincodeName) throws Exception {
```

Comment:
Because at least one of them is used in paradox (upgradeChaincode).  But java compiler does not see it because the usage is in separate project, and thus it produces warning.